src/Components/WorkflowCard/WorkflowCard.tsx

```diff
@@ -19,6 +19,7 @@ import WorkflowWarningButton from "Components/WorkflowWarningButton";
 import UpdateWorkflow from "./UpdateWorkflow";
 import WorkflowInputModalContent from "./WorkflowInputModalContent";
 import WorkflowRunModalContent from "./WorkflowRunModalContent";
+import cloneDeep from "clone-deep";
```

Comment:
I will check out the lodash clonedeep. I think I often avoid lodash because of how large of a package it is - but if we are already using it in this app, then we may as well leverage it

Reply:
I think the latest versions are treeshakeable, but I now try to do the import directly to the function which reduces the amount of code imported but I'd have to check to see how much it reduces the overall bundle size e.g.

`import cloneDeep from "lodash/cloneDeep"`